<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaveApproveWidget</class>
 <widget class="QWidget" name="LeaveApproveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDateTimeEdit" name="editStart"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="editEnd"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTextEdit" name="editReason"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tableHistory">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnSubmit">
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
